readthedocs/builds/admin.py

```diff
@@ -19,6 +25,32 @@
 from readthedocs.search.utils import _indexing_helper
 
 
+def _pretty_config(instance):
+    """
+    Function to display pretty version of our data
+
+    Thanks to PyDanny: https://www.pydanny.com/pretty-formatting-json-django-admin.html
+    """
+
+    # Convert the data to sorted, indented JSON
+    response = json.dumps(instance.config, sort_keys=True, indent=2)
+
+    # Truncate the data. Alter as needed
+    response = response[:5000]
```

Comment:
Wouldn't this give a _syntax_ error or something when applying the lexer?

Reply:
perhaps? I don't think we'll hit this, so we can probably just remove this line.